development-docs/DEV_GUIDE.md

```diff
@@ -59,13 +59,18 @@ To build Strimzi from a source the operator and Kafka code needs to be compiled
 
    This updates `060-Deployment-strimzi-cluster-operator.yaml`, replacing all the image references (in `image` and `value` properties) with ones with the same name but with the repository changed.
 
-5. Then you can deploy the Cluster Operator by running: 
 
-        kubectl create -f install/cluster-operator
+5. The installation files assume you're installing into the namespace `myproject`.  If you wish to use a different one, you'll need to replace it in the installation files.
```

Comment:
You should probably mention that this needs to be reverted before committing the changes / opening the PR.

Reply:
Done :)